modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx

```diff
@@ -118,17 +118,17 @@ const fieldSettingsMap = new Map<string, ObjectFieldSetting[]>([
 
 async function getFieldSettingsByBusinessType(
 	objectRelationshipId: number,
+	setListTypeDefinitions: (value: PickList[]) => void,
```

Comment:
@igor-franca Check if the Picklist interface was not renamed as well. In case it was not, please rename it.